commands/group_test.go

```diff
@@ -0,0 +1,152 @@
+package commands
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestChannelGroupEnableCmdF() {
+	s.Run("Enable group constrains with existing team and channel", func() {
+		teamArg := "team-id"
+		mockTeam := model.Team{Id: teamArg}
+		channelPart := "channel-id"
+		mockChannel := model.Channel{Id: channelPart}
+		channelArg := strings.Join([]string{teamArg, channelPart}, ":")
```

Comment:
nit: this could be simplified with:

```suggestion
		channelArg := teamArg + ":" + channelPart
```

Same for the rest of instances